### PR TITLE
add clux/kube-rs client

### PR DESCRIPTION
**Pull request by clux**
add [clux/kube-rs](https://github.com/clux/kube-rs) to list of [community-maintained client libraries](https://kubernetes.io/docs/reference/using-api/client-libraries/#community-maintained-client-libraries)

### CODE REVIEW

**Comment by clux** [comment]
Woah. Netlify on PR, this is swish. [This is the affected page from the netlify build](https://deploy-preview-14759--kubernetes-io-master-staging.netlify.com/docs/reference/using-api/client-libraries/#community-maintained-client-libraries)

File: content/en/docs/reference/using-api/client-libraries.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ their authors, not the Kubernetes team.
 | Ruby                 | [github.com/abonas/kubeclient](https://github.com/abonas/kubeclient) |
 | Ruby                 | [github.com/kontena/k8s-client](https://github.com/kontena/k8s-client) |
 | Rust                 | [github.com/ynqa/kubernetes-rust](https://github.com/ynqa/kubernetes-rust) |
+| Rust                 | [github.com/clux/kube-rs](https://github.com/clux/kube-rs) |
```
Comment (sftim) [inline content/en/docs/reference/using-api/client-libraries.md]: Hi @clux 
I think this should sort before `github.com/ynqa/kubernetes-rust`

No big deal though!
/lgtm

Comment (clux) [reply content/en/docs/reference/using-api/client-libraries.md]: Reordered it!

**Pull request merged**